src/XRP/issued-currency-client.ts

```diff
@@ -68,21 +68,41 @@ export default class IssuedCurrencyClient {
    * @see https://xrpl.org/trust-lines-and-issuing.html
    *
    * @param account The account for which to retrieve associated trust lines, encoded as an X-Address.
+   * @param peerAccount (Optional) The address of a second account. If provided, show only trust lines connecting the two accounts.
    * @see https://xrpaddress.info/
    * @returns An array of TrustLine objects, representing all trust lines associated with this account.
    */
-  public async getTrustLines(account: string): Promise<Array<TrustLine>> {
+  public async getTrustLines(
+    account: string,
+    peerAccount?: string,
+  ): Promise<Array<TrustLine>> {
```

Comment:
this seems reasonable. have you considered making a separate function for just a peer (ie, `getTrustLineWithPeer`)? 

i think i'm ok with this api but just wanted to throw that out there

Reply:
yeah I think i'm ok with this for now too.